docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -918,4 +918,89 @@ spec:
   platforms: macOS
   tags: compliance, CIS, CIS6.1.4
   platform: darwin
-  contributors: GuillaumeRoss
\ No newline at end of file
+  contributors: GuillaumeRoss
+
+---
+    apiVersion: v1
+    kind: policy
+    spec:
+      name: Ensure Windows Firewall - Domain Profile is enabled (Windows)
```

```suggestion
      name: Windows Firewall, Domain Profile enabled (Windows)
```

For most policies we're currently using a "[setting name] + [enabled/disable] + ([platform])" naming convention. This way, the policy name is brief and concise. For example, "Automatic update downloads enabled (macOS)."

When possible, I think it makes sense to use this convention.